catalog/app/containers/NavBar/Help.js

```diff
@@ -0,0 +1,331 @@
+}
+
+function ObjectKey() {
+  const classes = useStyles()
+  return <sup className={classes.sup}>+</sup>
+}
+
+function Code({ children }) {
+  const classes = useStyles()
+
+  return <code className={classes.code}>{children}</code>
+}
+
+function ItemSyntax({ item, namespace }) {
+  const { id, isObject, isPackage } = item
+
+  const syntaxI18nId = `${namespace}.${id}.syntax`
+
+  return (
+    <M.Typography variant="body2">
+      <Code>
+        <FM id={syntaxI18nId} />
+      </Code>
+      {isObject && <ObjectKey />}
+      {isPackage && <PackageKey />}
+    </M.Typography>
+  )
+}
+
+function ItemTitle({ item, namespace }) {
+  const { id } = item
+
+  const syntaxI18nId = `${namespace}.${id}.syntax`
+  const titleI18nId = `${namespace}.${id}.title`
+
+  return (
+    <M.Typography variant="body2">
+      <FM
+        id={titleI18nId}
+        values={{
+          syntax: (
+            <Code>
+              <FM id={syntaxI18nId} />
+            </Code>
+          ),
+        }}
+      />
+    </M.Typography>
+  )
+}
+
+function ItemExample({ item, namespace }) {
+  const { id } = item
+
+  const exampleI18nId = `${namespace}.${id}.example`
+
+  return (
+    <M.Typography variant="body2">
+      <Code>
+        <FM id={exampleI18nId} />
+      </Code>
+    </M.Typography>
+  )
+}
+
+function Item({ intl, item, namespace }) {
+  const t = M.useTheme()
+  const xs = M.useMediaQuery(t.breakpoints.down('xs'))
+
+  const exampleI18nId = `${namespace}.${item.id}.example`
+  const hasExample = intl.messages[exampleI18nId]
+  const titleI18nId = `${namespace}.${item.id}.title`
+  const hasTitle = intl.messages[titleI18nId]
+
+  return (
+    <M.Grid container>
+      <M.Grid item xs={xs ? 4 : 3}>
+        <ItemSyntax item={item} namespace={namespace} />
+      </M.Grid>
+      {hasTitle && (
+        <M.Grid item xs>
+          <ItemTitle item={item} namespace={namespace} />
+        </M.Grid>
+      )}
+      {hasExample && (
+        <M.Grid item xs align="right">
+          <ItemExample item={item} namespace={namespace} />
+        </M.Grid>
+      )}
+    </M.Grid>
+  )
+}
+
+const ItemWrapper = injectIntl(Item)
+
+function Legend() {
+  const classes = useStyles()
+
+  return (
+    <M.Box className={classes.caption}>
+      <M.Typography variant="caption">
+        <FM id="searchQuerySyntax.isPackage" values={{ key: <PackageKey /> }} />
+      </M.Typography>
+      ,{' '}
+      <M.Typography variant="caption">
+        <FM id="searchQuerySyntax.isObject" values={{ key: <ObjectKey /> }} />
+      </M.Typography>
+    </M.Box>
+  )
+}
+
+function DocsExternalLink() {
+  const classes = useStyles()
+
+  const ES_V = '6.7'
+  const ES_REF = `https://www.elastic.co/guide/en/elasticsearch/reference/${ES_V}/query-dsl-query-string-query.html#query-string-syntax`
+
+  const link = (
+    <StyledLink href={ES_REF}>
+      <FM id="searchQuerySyntax.captionLink" />
+    </StyledLink>
+  )
+
+  return (
+    <M.Box className={classes.caption}>
```

Comment:
i think we can use a plain `div` here since we're not using any of the `Box`'s features